packages/core/test/test-util/test-services/v4/test-service/TestEntity.ts

```diff
@@ -115,6 +115,16 @@ export class TestEntity extends Entity implements TestEntityType {
    * @nullable
    */
   complexTypeProperty?: TestComplexType;
+  /**
+   * Collection Property with string inside.
+   * @nullable
+   */
+  collectionPropertyWithString?: string[];
+  /**
+   * Collection Property with complex type inside.
+   * @nullable
+   */
+  collectionPropertyWithComplexType?: TestComplexType[];
```

Comment:
Is this a manual addition? If yes, please add it to either the generate-test-services script or the VDM somehow, otherwise it will disappear on regeneration and cause problems in the future.